index.js

```diff
@@ -4,6 +4,12 @@ module.exports = function buildAirbnbPreset(context, options) {
   var preset; // eslint-disable-line no-var
   if (options && options.modules === false) {
     preset = require('babel-preset-es2015').buildPreset(null, { modules: false });
+  } else if (options && options.node === true) {
```

Comment:
i don't think this should be a boolean - i think it should be a version number.

That way we don't need to bump the preset later when we upgrade node.